### DISCUSSION TITLE
Welcome to AlgorithmsAndDataStructuresInAction Discussions!

### DISCUSSION THREAD

**Discussion by mlarocca** [mlarocca/AlgorithmsAndDataStructuresInAction#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VirtualZen** [comment]
I'm pulling my hairs off. cant run or compile the samples. need a dummies set of steps.

**Reply by mlarocca** [reply]
Which language? Java?

**Reply by VirtualZen** [reply]
I tried Java, with Jetbrains IDE.

Also I tried JavaScript with the instructions here in GitHub but I can’t go beyond the nvm install command. But that seems an unrelated issue.

I noticed the Java version doesn’t have setup instructions like the JavaScript page.

**Reply by VirtualZen** [reply]
These are the JavaScript steps:

nvm install stable - doesn’t do anything
npm install - seems to work
After that I tried the nvm line, even used homebrew.
